src/Web/Sfa.Das.Sas.Shared.Components/Views/Shared/Components/TrainingProvider/SearchResults/Default.cshtml

```diff
@@ -22,10 +22,6 @@
         <input type="hidden" asp-for="@Model.SearchQuery.Page" />
         <input type="hidden" asp-for="@Model.SearchQuery.Postcode" />
         <input type="hidden" asp-for="@Model.SearchQuery.SortOrder" />
```

Comment:
Does HasNationalProviders need adding to the hidden inputs?

Reply:
Don't think so. The flag for the search parameter hasn't changed, just the was we display the filter has changed.